test/martha/martha_v3.test.js

```diff
@@ -354,25 +351,8 @@ test.serial('martha_v3 calls the correct endpoints when only the fileName is req
     );
     t.is(response.statusCode, 200);
     const result = response.send.lastCall.args[0];
-    sinon.assert.callCount(getJsonFromApiStub, 3); // Bond was not called to retrieve the googleServiceAccount
-    t.deepEqual(
-        { ...result },
-        mask(bdcDrsMarthaResult(googleSAKeyObject, drsAccessUrlResponse), 'fileName'),
-    );
-
-    const requestedBondAccessTokenUrl = getJsonFromApiStub.getCall(1).args[0];
-    const accessTokenUrlMatches = requestedBondAccessTokenUrl.match(bondAccessTokenUrlRegEx);
-    t.truthy(accessTokenUrlMatches, 'Bond SA key URL called does not match Bond SA key URL regular expression');
-    const expectedAccessTokenProvider = 'fence';
-    const actualAccessTokenProvider = accessTokenUrlMatches[2];
-    t.is(actualAccessTokenProvider, expectedAccessTokenProvider);
-
-    t.is(
-        getJsonFromApiStub.getCall(2).args[0],
-        `https://${config.HOST_BIODATA_CATALYST_STAGING}/ga4gh/drs/v1/objects` +
-        '/dg.712C/fa640b0e-9779-452f-99a6-16d833d15bd0/access/gs',
-    );
-    t.is(getJsonFromApiStub.getCall(2).args[1], `Bearer ${bondAccessTokenResponse.token}`);
+    sinon.assert.callCount(getJsonFromApiStub, 1); // Bond was not called to retrieve the googleServiceAccount
+    t.deepEqual({ ...result }, { fileName: null });
```

Comment:
TOL is `{ ...result }` not the same as `result`?

Reply:
In this case, probably. There were (are?) similar test cases where `deepEqual` tested the [equality of the wrapper](https://github.com/avajs/ava/blob/main/docs/03-assertions.md#deepequalvalue-expected-message).

For consistency and because a lot of these lines are later copy-pasted I used the same "erase the wrapper type" syntax here.